uio/uiotest/uiotest.go

```diff
@@ -1,3 +1,5 @@
+// package uiotest contains tests for uio functions.
```

Comment:
package comments go right next to the `package` statement, not above the copyright

Reply:
Done.

...and I need Cap the 'P'